src/index.html

```diff
@@ -20,8 +20,407 @@
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
+            </div>
+            <a
+              class="button-link"
+              href="buy"
+              data-qa="card-hover"
+            >
+              <button
+                class="button"
+                data-qa="card-hover"
+              >
+                BUY
+              </button>
+            </a>
+          </div>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="image"
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <div class="content-wrapper">
+            <div class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="product-code">Product code: 195434</div>
+            <div class="content-revie">
+              <div class="stars">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+              </div>
+              <div class="revie-text">Reviews: 5</div>
+            </div>
+            <div class="content-price">
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
+            </div>
+            <a
+              class="button-link"
+              href="buy"
+              data-qa="hover"
+            >
+              <button class="button">BUY</button>
+            </a>
+          </div>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="image"
+            src="images/imac.jpeg"
+            alt="imac"
+          />
+          <div class="content-wrapper">
+            <div class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="product-code">Product code: 195434</div>
+            <div class="content-revie">
+              <div class="stars">
+                <div class="stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
+              </div>
+              <div class="revie-text">Reviews: 5</div>
+            </div>
+            <div class="content-price">
+              <div class="price-text">Price:</div>
+              <div class="prise-num">$2,199</div>
+            </div>
+            <a
+              class="button-link"
+              href="buy"
+              data-qa="hover"
```

Comment:
The `data-qa` attribute is inconsistently used here as `hover`. Ensure consistency across your HTML file as per the task requirements.